refresh-data.sh

```diff
@@ -42,7 +42,7 @@ if [[ -z "$refresh_dump_name" ]]; then
 
 Or you can define the location of a dump and this script will download it for you:
 
-	export CFGOV_PROD_DB_LOCATION=http://some-bucket.s3.amazonaws.com/wherever/production_django.sql.gz
+	export CFGOV_PROD_DB_LOCATION=https://some-bucket.s3.amazonaws.com/wherever/production_django.sql.gz
```

Comment:
This won't work with HTTPS if you include the s3.amazonaws.com (see above).

Reply:
Removed in https://github.com/cfpb/cfgov-refresh/pull/3998/commits/225af8f9afe086abf783200f5b402adc7e1a1a7c